### PR TITLE
Setup Sun* CI

### PR DESCRIPTION
**Pull request by vunt-3611**
## Related Tickets
- [#82569](https://edu-redmine.sun-asterisk.vn/issues/82569)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- Clone code từ rails tutorial về, sau đó làm theo các bước ở PJ1

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by vunt-3611** [comment]
ready

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# MySQL. Versions 5.5.8 and up are supported.
```
Comment (ghost) [inline config/database.yml]: ignore file này đi em

File: config/credentials.yml.enc [hunk 2]
```diff
@@ -0,0 +1 @@
+8y568Ee3rnhfXg9fA1GalwyHE35RjXh6QTKWMaHYwNMwKBKlfGEVId1uCc910C2eROdDsqbeMe+LRzztNV5RMsN0DiEPxf9zSVeyG410M6ltgQvl+B/JrddklMBRKdrEATSQ5nQITZ/dp9ENPYQkMmQF+15PeRONu3/2FVw4DKyoHVVF9G2s95VperTUEUZpu7gz5eUxKa5vk6K1r8bd659L/z3hV4H473ezKoasSnOBE0E/qubyy9iIEq5XGwvApR5UCb/aQ9kJBTNO23MOXtFHZStFFwb5sCPIsnLOFJM2tNxPEChUtlmZhwcvaSqXDu8fBLNeBWpapdJM723u7YEQLzxqY/ZwciUpV5j5gXhvzYe0Ym22kFvHosL+6Z5qGKP/Dz1Acf9HEFtOjGGc7NxwpdxLCYQFHm6/--dsAjZBWWnfvc/c1k--Iecfs1nfNsODaFl5nAIcBg==
```
Comment (ghost) [inline config/credentials.yml.enc]: ignore file này đi em

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+ruby "3.2.2"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.0.5"
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem "sprockets-rails"
+
+# Use mysql as the database for Active Record
```
Comment (ghost) [inline Gemfile]: Xóa hết cmt đi nhé

**Review by ghost — COMMENTED** [review]
cmt

**Comment by vunt-3611** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**